ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py

```diff
@@ -51,7 +51,7 @@ def setUp(self):
         self.standby_tor_ip = self.test_params['standby_tor_ip']
         self.ptf_portchannel_indices = self.test_params['ptf_portchannel_indices']
         self.indice_to_portchannel = {}
-        for port_channel, indices in self.ptf_portchannel_indices.items():
+        for port_channel, indices in list(self.ptf_portchannel_indices.items()):
```

Comment:
There is no need to make the iterable(`self.ptf_portchannel_indices.items()`) to be a list to make the loop work.

Reply:
Updated accordingly.